js2-mode.el

```diff
@@ -9913,7 +9915,44 @@ Returns the list in reverse order.  Consumes the right-paren token."
       (js2-must-match js2-RP "msg.no.paren.arg")
       result)))
 
-(defun js2-parse-member-expr (&optional allow-call-syntax)
+(defun js2-parse-lhs-expr ()
+  "Parse a `LeftHandSideExpression'.
+
+The three possible productions are collectively handled like a
+`CallExpression' with optional `Arguments' at end.  Unary bind is
+handled specially because it is the only one not beginning with a
+`NewExpression'."
+  (let ((pn (if (= (js2-current-token-type) js2-COLONCOLON)
```

Comment:
That's correct. Not sure if it's easy or beneficial to make this aspect more consistent.